tools/migrate/src/main.rs

```diff
@@ -155,6 +163,8 @@ fn main() -> anyhow::Result<()> {
 
     dbg!(&src_dir, &dst_dir);
 
+    let ignore_broken_rows = matches.is_present("ignore_broken_rows");
```

Comment:
Isn't it missing adding this to a Config and passing it as third argument to the `Database::new` below?

Reply:
yes, sorry, it probably did not compile because of that

Reply:
should be fixed now, please download and compile to confirm, macos seems to be struggling with rocksdb :/